apps/extension/src/Setup/AccountCreation/AccountCreation.tsx

```diff
@@ -1,204 +1,43 @@
-import { AccountCreationRoute, accountCreationSteps } from "./types";
-
 import { Icon, IconName, IconSize } from "@anoma/components";
-import {
-  Start,
-  Password,
-  SeedPhrase,
-  SeedPhraseConfirmation,
-  Completion,
-} from "./Steps";
 import {
   AccountCreationContainer,
   TopSection,
   TopSectionHeaderContainer,
   TopSectionButtonContainer,
   RouteContainer,
-  MotionContainer,
 } from "./AccountCreation.components";
-import { AccountCreationDetails } from "Setup/AccountCreation/types";
-
-type AnimatedTransitionProps = {
-  elementKey: string;
-  children: JSX.Element;
-};
-
-/**
- * This is a utility to facilitate the animated transitions.
- */
-const AnimatedTransition: React.FC<AnimatedTransitionProps> = (props) => {
-  const { children, elementKey } = props;
-  return (
-    <MotionContainer
-      key={elementKey}
-      initial={{ opacity: 0 }}
-      animate={{ opacity: 1 }}
-      exit={{ opacity: 0 }}
-    >
-      {children}
-    </MotionContainer>
-  );
-};
-
-type Props = {
-  requester: ExtensionRequester;
-};
 
 /**
  * The main purpose of this is to coordinate the flow for creating a new account.
  * It persists the data between the screens in the flow.
  */
-const AccountCreation: React.FC<Props> = ({ requester }) => {
-  const [accountCreationDetails, setAccountCreationDetails] =
-    useState<AccountCreationDetails>({
-      alias: "",
-    });
-  const [seedPhrase, setSeedPhrase] = useState<string[]>();
-  const [stepIndex, setStepIndex] = useState(0);
-
+const AccountCreation: React.FC = () => {
   const themeContext = useContext(ThemeContext);
   const navigate = useNavigate();
 
-  const location = useLocation();
-
-  // info for disabling the back button in the last step
-  const isLastIndex = accountCreationSteps.length - 1 === stepIndex;
-
-  useEffect(() => {
-    // at the load we redirect to the first step
-    // this way we do not need to expose the flow routes to outside
-    navigate(AccountCreationRoute.Start);
-  }, []);
-
-  const navigateToNext = (): void => {
-    setStepIndex((stepIndex) => stepIndex + 1);
-    navigate(`${accountCreationSteps[stepIndex + 1]}`);
-  };
-
-  const navigateToPrevious = (): void => {
-    setStepIndex((stepIndex) => {
-      return stepIndex - 1;
-    });
-    navigate(`${accountCreationSteps[stepIndex - 1]}`);
-  };
-
   return (
     <AccountCreationContainer>
       <TopSection>
         <TopSectionButtonContainer>
-          {!isLastIndex && stepIndex !== 0 && (
-            <a
-              onClick={() => {
-                navigateToPrevious();
-              }}
-              style={{ cursor: "pointer" }}
-            >
-              <Icon
-                iconName={IconName.ChevronLeft}
-                strokeColorOverride={themeContext.colors.utility2.main60}
-                iconSize={IconSize.L}
-              />
-            </a>
-          )}
+          <a onClick={() => navigate(-1)} style={{ cursor: "pointer" }}>
```

Comment:
The chevron is moving about a bit between pages. I don't think it needs to block the PR - maybe we could add it as a new issue and do it when doing other small UI fixes?

Reply:
Yeah, it's still a little buggy UI-wise. I can clean that up on this PR. I'm not focusing too much on UI, but it should still be acceptable. I'll be building out the flow in the next PR, and we'll also have to redo it once we get final designs

Reply:
I've decided to rework the components across the `Setup` app to remove all of the duplication that's happening, and should get this issue fixed!

Reply:
Resolving this as it should all be fixed now